plugins/home/src/homePageComponents/StarredEntities/Content.tsx

```diff
@@ -41,7 +41,12 @@ import useAsync from 'react-use/lib/useAsync';
  *
  * @public
  */
-export const Content = () => {
+
+interface starredEntitiesProp {
```

Comment:
Types always start with an uppercase letter. But in this case I'd say just don't make a name for it at all. Just put it verbatim on the component below instead.

```ts
export const Content = (props: { emptyState: JSX.Element }) => {
```

or something like that. Then you can put whatever there.

This feature isn't even exported to the public - nobody will ever be able to make it show anything else than "Click the star beside an entity name to add it to this list!'". But I don't mind too much about that extra hidden ability in here